cmd/sovereignnode/main.go

```diff
@@ -239,6 +240,13 @@ func readConfigs(ctx *cli.Context, log logger.Logger) (*sovereignConfig.Sovereig
 	}
 	log.Debug("config", "file", sovereignExtraConfigPath)
 
+	configurationPaths.Epoch = ctx.GlobalString(epochConfigurationFile.Name)
+	sovereignEpochConfig, err := sovereignConfig.LoadSovereignEpochConfig(configurationPaths.Epoch)
```

Comment:
Why did you add these?
These are already loaded within enableEpochs file.

Reply:
This is different. It's loading SovereignEpochConfig struct, not the EpochConfig